applications/openshift/networking/var_network_policies_namespaces_whitelist_regex.var

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+title: 'Whitelist Namespaces for Network Policies'
+
+description: |-
+    Namespaces regular expression explicitly allowed
+    through network policy filters, e.g. setting value to
+    "namespace1|namespace2" will whitelist namespace
+    "namespace1" and "namespace2" for network policies checks.
+
+type: string
+
+operator: equals
+
+interactive: false
```

Comment:
I guess this field is not consumed by CO, but if this should be editable by the user, this should be `true`
```suggestion
interactive: true
```